src/psyclone/psyir/nodes/call.py

```diff
@@ -127,6 +245,31 @@ def routine(self):
         '''
         return self._routine
 
+    @property
+    def named_args(self):
+        '''
+        :returns: a list containing the names of named arguments. If the \
```

Comment:
I am planning to check that an argument name conforms to some basic rule (probably [a-zA-Z][a-zA-Z0-9_]*). This would rule out empty string, so have not updated the words to include "".